lib/db2.js

```diff
@@ -52,15 +52,27 @@ util.inherits(DB2, IBMDB);
  * @param {Function} [callback] The callback function
  */
 DB2.prototype.create = function(model, data, options, callback) {
+  debug('DB2.prototype.create: model=%j, data=%j, options=%j',
+        model, options);
```

Comment:
On the second thought it should be:
```
debug('DB2.prototype.create: model=%s, data=%j, options=%j',
  model, data, options);
```

Please note `%s` for model; Please double check; IIRC `model` is string not an object